x/auth/client/cli/broadcast.go

```diff
@@ -11,6 +11,9 @@ import (
 	authclient "github.com/cosmos/cosmos-sdk/x/auth/client"
 )
 
+// TODO HV2: GetBroadcastCommand is slightly different in heimdall.
+//  We need import the `helper` and apply the changes from heimdall's `client/tx/broadcast.go`
+
```

Comment:
Same thoughts here, unless this was already discussed.

Reply:
Thanks, same answer as before